template.yaml

```diff
@@ -374,13 +374,14 @@ Resources:
   CreateUploadFunction:
     Type: AWS::Serverless::Function
     Properties:
+      CodeUri: create-upload
       Environment:
         Variables:
           S3_UPLOAD_BUCKET: !Ref  S3UploadBucket
           ALLOWED_ORIGIN:  '*'
       Handler: no.unit.nva.fileupload.CreateUploadHandler::handleRequest
       Runtime: java11
-      MemorySize: 1400
+      MemorySize: 1408
```

Comment:
why such number?

Reply:
CodeGuru complained in an earlier PR, but on a second look 1408 is no better than 1400. I'll see if I can find the message again.